internal/cli/atlas_alerts_acknowledge.go

```diff
@@ -54,12 +54,15 @@ func (opts *atlasAlertsAcknowledgeOpts) Run() error {
 
 func (opts *atlasAlertsAcknowledgeOpts) newAcknowledgeRequest() *atlas.AcknowledgeRequest {
 	// To acknowledge an alert “forever”, set the field value to 100 years in the future.
+	until := &opts.until
 	if opts.forever {
-		opts.until = time.Now().AddDate(100, 1, 1).Format(time.RFC3339)
+		foreverDate := time.Now().AddDate(100, 1, 1).Format(time.RFC3339)
+		until = &foreverDate
+	} else if *until == "" {
+		until = nil
```

Comment:
one way you can simplify this by a lot is to change the type of until in `atlasAlertsAcknowledgeOpts` to a pointer ;)